nix/compiler_rt.patch

```diff
@@ -150,47 +150,47 @@ index 6f75baa7e354..dbe1e69487b1 100644
    if(SANITIZER_USE_SYMBOLS)
      add_sanitizer_rt_symbols(${hwasan_runtime}
 diff --git a/compiler-rt/lib/scudo/standalone/CMakeLists.txt b/compiler-rt/lib/scudo/standalone/CMakeLists.txt
-index 60092005cc33..72c2b83a2dad 100644
+index dc700cec9bec..112d8d77c3ed 100644
 --- a/compiler-rt/lib/scudo/standalone/CMakeLists.txt
 +++ b/compiler-rt/lib/scudo/standalone/CMakeLists.txt
-@@ -232,19 +232,19 @@ if(COMPILER_RT_HAS_SCUDO_STANDALONE)
-     DEPS ${SCUDO_DEPS}
-     PARENT_TARGET scudo_standalone)
+@@ -232,19 +232,19 @@ add_compiler_rt_runtime(clang_rt.scudo_standalone_cxx
+   DEPS ${SCUDO_DEPS}
+   PARENT_TARGET scudo_standalone)
  
--  if(COMPILER_RT_SCUDO_STANDALONE_BUILD_SHARED)
--    add_compiler_rt_runtime(clang_rt.scudo_standalone
--      SHARED
--      ARCHS ${SCUDO_STANDALONE_SUPPORTED_ARCH}
--      SOURCES ${SCUDO_SOURCES} ${SCUDO_SOURCES_C_WRAPPERS} ${SCUDO_SOURCES_CXX_WRAPPERS}
--      ADDITIONAL_HEADERS ${SCUDO_HEADERS}
--      CFLAGS ${SCUDO_CFLAGS}
--      DEPS ${SCUDO_DEPS}
--      OBJECT_LIBS ${SCUDO_OBJECT_LIBS}
--      LINK_FLAGS ${SCUDO_LINK_FLAGS}
--      LINK_LIBS ${SCUDO_LINK_LIBS}
--      PARENT_TARGET scudo_standalone)
--  endif()
-+  # if(COMPILER_RT_SCUDO_STANDALONE_BUILD_SHARED)
-+  #   add_compiler_rt_runtime(clang_rt.scudo_standalone
-+  #     SHARED
-+  #     ARCHS ${SCUDO_STANDALONE_SUPPORTED_ARCH}
-+  #     SOURCES ${SCUDO_SOURCES} ${SCUDO_SOURCES_C_WRAPPERS} ${SCUDO_SOURCES_CXX_WRAPPERS}
-+  #     ADDITIONAL_HEADERS ${SCUDO_HEADERS}
-+  #     CFLAGS ${SCUDO_CFLAGS}
-+  #     DEPS ${SCUDO_DEPS}
-+  #     OBJECT_LIBS ${SCUDO_OBJECT_LIBS}
-+  #     LINK_FLAGS ${SCUDO_LINK_FLAGS}
-+  #     LINK_LIBS ${SCUDO_LINK_LIBS}
-+  #     PARENT_TARGET scudo_standalone)
-+  # endif()
+-if(COMPILER_RT_SCUDO_STANDALONE_BUILD_SHARED)
+-  add_compiler_rt_runtime(clang_rt.scudo_standalone
+-    SHARED
+-    ARCHS ${SCUDO_STANDALONE_SUPPORTED_ARCH}
+-    SOURCES ${SCUDO_SOURCES} ${SCUDO_SOURCES_C_WRAPPERS} ${SCUDO_SOURCES_CXX_WRAPPERS}
+-    ADDITIONAL_HEADERS ${SCUDO_HEADERS}
+-    CFLAGS ${SCUDO_CFLAGS}
+-    DEPS ${SCUDO_DEPS}
+-    OBJECT_LIBS ${SCUDO_OBJECT_LIBS}
+-    LINK_FLAGS ${SCUDO_LINK_FLAGS}
+-    LINK_LIBS ${SCUDO_LINK_LIBS}
+-    PARENT_TARGET scudo_standalone)
+-endif()
++# if(COMPILER_RT_SCUDO_STANDALONE_BUILD_SHARED)
++#   add_compiler_rt_runtime(clang_rt.scudo_standalone
++#     SHARED
++#     ARCHS ${SCUDO_STANDALONE_SUPPORTED_ARCH}
++#     SOURCES ${SCUDO_SOURCES} ${SCUDO_SOURCES_C_WRAPPERS} ${SCUDO_SOURCES_CXX_WRAPPERS}
++#     ADDITIONAL_HEADERS ${SCUDO_HEADERS}
++#     CFLAGS ${SCUDO_CFLAGS}
++#     DEPS ${SCUDO_DEPS}
++#     OBJECT_LIBS ${SCUDO_OBJECT_LIBS}
++#     LINK_FLAGS ${SCUDO_LINK_FLAGS}
++#     LINK_LIBS ${SCUDO_LINK_LIBS}
++#     PARENT_TARGET scudo_standalone)
++# endif()
```

Comment:
Do we need these lines anymore? If so, should we leave a note about why they are currently commented out?

Reply:
It's already explained here: https://github.com/spaceandtimelabs/blitzar/blob/main/nix/clang.nix#L30-L31